lib/rspec/core/output_wrapper.rb

```diff
@@ -3,17 +3,25 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        output.respond_to?(name, priv)
+      end
+
+      def method_missing(name, *args, &block)
+        output.method_missing(name, *args, &block)
+      end
```

Comment:
OK, I can see there's more complexity here I haven't been aware of.

Let's go with what you have (once you get it green), but can you leave a `TODO` command to revisit?  I think it's quite confusing to use both `define_method` for all intended proxy methods and also use `method_missing`.